[2021-08-21 22:43:14,783] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: weather_prediction_airflow.weather_airflow 2021-08-22T02:36:25.496443+00:00 [queued]>
[2021-08-21 22:43:14,787] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: weather_prediction_airflow.weather_airflow 2021-08-22T02:36:25.496443+00:00 [queued]>
[2021-08-21 22:43:14,787] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-08-21 22:43:14,787] {taskinstance.py:1088} INFO - Starting attempt 1 of 2
[2021-08-21 22:43:14,788] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-08-21 22:43:14,793] {taskinstance.py:1107} INFO - Executing <Task(BashOperator): weather_airflow> on 2021-08-22T02:36:25.496443+00:00
[2021-08-21 22:43:14,797] {standard_task_runner.py:52} INFO - Started process 27770 to run task
[2021-08-21 22:43:14,807] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'weather_prediction_airflow', 'weather_airflow', '2021-08-22T02:36:25.496443+00:00', '--job-id', '66', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/Airflow.py', '--cfg-path', '/var/folders/kh/c93m3t_d2rgcvcbylylqtx240000gn/T/tmpnow6ph_k', '--error-file', '/var/folders/kh/c93m3t_d2rgcvcbylylqtx240000gn/T/tmpz_198kae']
[2021-08-21 22:43:14,809] {standard_task_runner.py:77} INFO - Job 66: Subtask weather_airflow
[2021-08-21 22:43:14,860] {logging_mixin.py:104} INFO - Running <TaskInstance: weather_prediction_airflow.weather_airflow 2021-08-22T02:36:25.496443+00:00 [running]> on host 1.0.0.127.in-addr.arpa
[2021-08-21 22:43:14,893] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=weather_prediction_airflow
AIRFLOW_CTX_TASK_ID=weather_airflow
AIRFLOW_CTX_EXECUTION_DATE=2021-08-22T02:36:25.496443+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-08-22T02:36:25.496443+00:00
[2021-08-21 22:43:14,894] {subprocess.py:52} INFO - Tmp dir root location: 
 /var/folders/kh/c93m3t_d2rgcvcbylylqtx240000gn/T
[2021-08-21 22:43:14,895] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'python /Users/kennethlau/Desktop/programing_exercise/side_projects/weather_prediction/a_pipeline.py']
[2021-08-21 22:43:14,905] {subprocess.py:74} INFO - Output:
[2021-08-21 22:44:25,592] {subprocess.py:78} INFO - RMSE: 1.853
[2021-08-21 22:44:25,793] {subprocess.py:82} INFO - Command exited with return code 0
[2021-08-21 22:44:25,840] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=weather_prediction_airflow, task_id=weather_airflow, execution_date=20210822T023625, start_date=20210822T024314, end_date=20210822T024425
[2021-08-21 22:44:25,863] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-08-21 22:44:25,892] {local_task_job.py:149} INFO - Task exited with return code 0
